services/src/main/java/org/keycloak/services/resources/account/AccountFormService.java

```diff
@@ -971,7 +971,7 @@ public static boolean isPasswordSet(KeycloakSession session, RealmModel realm, U
             if (referrerUri != null) {
                 referrerUri = RedirectUtils.verifyRedirectUri(session.getContext().getUri(), referrerUri, realm, referrerClient);
             } else {
-                referrerUri = ResolveRelative.resolveRelativeUri(session.getContext().getUri().getRequestUri(), client.getRootUrl(), referrerClient.getBaseUrl());
+                referrerUri = ResolveRelative.resolveRelativeUri(session.getContext().getUri().getRequestUri(), referrerClient.getRootUrl(), referrerClient.getBaseUrl());
             }
```

Comment:
This line means:
referrerUri = _Root URL_ of the built-in "_account_" client  + _Base URL_ of UMA client

It should be changed to:
referrerUri = _Root URL_ + _Base URL_ of UMA client